internals/link.py

```diff
@@ -0,0 +1,93 @@
+import logging
+from typing import Any
+from internals.core_models import FeatureEntry
+from internals.link_helpers import Link
+
+from google.cloud import ndb  # type: ignore
+
+
+class FeatureLinks(ndb.Model):
```

Comment:
It would be good to add a comment here to explain how and when the `FeatureLinks` entities are used.   E.g., "This information is used to display a preview of information from the linked page, which often saves the user from needing to click.".